content/docs/command-reference/status.md

```diff
@@ -19,45 +19,49 @@ positional arguments:
+
+| Mode   | Option            | Description                                                                                                                 |
+| ------ | ----------------- | --------------------------------------------------------------------------------------------------------------------------- |
+| local  | _none_            | Comparisons are made between data files in the workspace and corresponding files in the cache directory (e.g. `.dvc/cache`) |
+| remote | `--remote` (`-r`) | Comparisons are made between the cache, and a given DVC remote.                                                             |
+| remote | `--cloud` (`-c`)  | Comparisons are made between the cache, and the [default remote].                                                           |
 
 Without arguments, this command checks all `dvc.yaml` and `.dvc` files to
-rebuild and validate pipeline(s). It then compares their <abbr>outputs</abbr>
-against the actual data files or directories in the workspace. Any `targets`
-given to this command limit what to show changes for. It accepts paths to
-tracked files or directories (including paths inside tracked directories),
-`.dvc` files, and stage names (found in `dvc.yaml`).
+rebuild and validate pipeline(s). It then compares <abbr>outputs</abbr> defined
+in these files against the actual data in the workspace.
+
+Any `targets` given to this command limit what to show changes for. It accepts
+paths to tracked files or directories (including paths inside tracked
+directories), `.dvc` files, and stage names (found in `dvc.yaml`).
 
 The `--all-branches`, `--all-tags`, and `--all-commits` options enable comparing
 DVC-tracked files referenced in multiple Git commits at once.
 
 If no differences are detected, `dvc status` prints
-`Data and pipelines are up to date.` or
+`Data and pipelines are up to date`, or
 `Cache and remote 'myremote' are in sync` (if using the `-c` or `-r` options are
 used). If differences are detected, the changes in <abbr>dependencies</abbr>
-and/or <abbr>outputs</abbr> for each stage that differs are listed. For each
-item listed, either the file name or hash is shown, along with a _state
-description_, as detailed below:
+and/or <abbr>outputs</abbr> for each stage are listed. For each item listed,
+either the file name or hash is shown, along with a state description, as
+detailed bellow.
+
+[remote storage]: /doc/command-reference/remote
+[default remote]: /doc/command-reference/remote/default
 
 ### Local workspace status
 
 - _changed checksum_ means that the `.dvc` file hash has changed (e.g. someone
   manually edited it).
 
 - _always changed_ means that this is a `.dvc` file with no dependencies (see
-  `dvc add`) or that the stage in `dvc.yaml` has the `always_changed: true`
-  value set (see `--always-changed` option in `dvc stage add`).
+  `dvc add`) or that this stage (in `dvc.yaml`) has the `always_changed` field
+  set to `true` (see also `dvc stage add --always-changed`).
```

Comment:
@daavoo do we need to say also "or outputs" here like in https://github.com/iterative/dvc.org/pull/3286#pullrequestreview-883760555 ?

Reply:
I'm a little confused by the phrase `this is a `.dvc` file with no dependencies`. 

I think it might be outdated. It's not currently possible to have a `.dvc` file with dependencies, at all.